canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -61,10 +61,10 @@ $toolbar-divider-width: 1px;
 		.toolbar-item-content {
 			height: $toolbar-button-sm-height;
 			align-items: center;
-			min-width: 32px;
+			min-width: $spacing-07;
 
 			&.is-in-menu {
-				padding-left: 10px;
+				padding-left: $spacing-04;
```

Comment:
Can you double check what this looks like now compared to master? If off by alot, then revert